lua/autorun/server/sv_cfc_restart_lib.moon

```diff
@@ -0,0 +1,27 @@
+RestartUrl = file.Read "cfc/restart/url.txt", "DATA"
+RestartUrl = string.Replace RestartUrl, "\r", ""
+RestartUrl = string.Replace RestartUrl, "\n", ""
+
+getRestartToken = -> file.Read "cfc/restart/token.txt", "DATA"
+
+restart = (sucess, failure) -> http.Post RestartUrl, { restartToken: getRestartToken! }, success, failure
+
+generateTimerName = -> math.Round(math.Random! * 1000)
```

Comment:
Thinking more on this: You might want to utilize the info provided in `debug.getinfo` because it has the name of the calling function (**beware gotcha**: name is nil if anon function is caller) and the line number and such.

Could be useful for registering the timer and the webhook and such.